gui/src-tauri/src/main.rs

```diff
@@ -64,6 +63,14 @@ fn get_launch_path(cli: Cli) -> Option<PathBuf> {
 		}
 	}
 
+	// This is only for AppImage
+	if let Some(appimage) = env::var_os("APPDIR") {
+		let path = PathBuf::from(appimage);
+		if is_valid_path(&path) {
```

Comment:
change `is_valid_path(path: &PathBuf)` to `is_valid_path(path: &Path)`. No need to have a reference to an owned type when the ref type is better.

Reply:
``is_valid_path`` returns the path itself, we need to own it.